# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ctwm/manpage.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./ctwm.man.orig	2001-12-11 16:38:52.000000000 +0100
+++ ./ctwm.man	2003-04-26 20:06:36.000000000 +0200
@@ -1,13 +1,10 @@
 .de EX		\"Begin example
-.ne 5
 .if n .sp 1
 .if t .sp .5
 .nf
-.in +.5i
 ..
 .de EE
 .fi
-.in -.5i
 .if n .sp 1
 .if t .sp .5
 ..
